aries_vcx/tests/test_did_exchange.rs

```diff
@@ -0,0 +1,99 @@
+#[macro_use]
+extern crate log;
+mod fixtures;
+mod utils;
+
+use aries_vcx::protocols::did_exchange::service::requester::{
+    ConstructRequestConfig, DidExchangeServiceRequester, PairwiseConstructRequestConfig,
+};
+use aries_vcx::protocols::did_exchange::service::responder::{DidExchangeServiceResponder, ReceiveRequestConfig};
+use aries_vcx::protocols::did_exchange::states::requester::request_sent::RequestSent;
+use aries_vcx::protocols::did_exchange::states::responder::response_sent::ResponseSent;
+use aries_vcx::protocols::did_exchange::transition::transition_result::TransitionResult;
+use aries_vcx::utils::devsetup::SetupPoolDirectory;
+use did_doc::schema::verification_method::PublicKeyField;
+use did_doc_sov::extra_fields::didcommv2::ExtraFieldsDidCommV2;
+use did_doc_sov::service::didcommv2::ServiceDidCommV2;
+use did_doc_sov::service::ServiceSov;
+use messages::msg_fields::protocols::out_of_band::invitation::Invitation;
+use url::Url;
+
+use crate::utils::devsetup_alice::create_alice;
+use crate::utils::devsetup_faber::create_faber;
+
+#[tokio::test]
+async fn did_exchange_test() {
```

Comment:
currently failing on line `47` (even before the `todo!`)

Reply:
I know...